From felipe.balbi@nokia.com  Mon Dec  7 16:41:56 2009
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Fri,  4 Dec 2009 15:47:42 +0200
Subject: USB: core: fix sparse warning for static function
To: linux-usb@vger.kernel.org
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Felipe Balbi <felipe.balbi@nokia.com>
Message-ID: <1259934467-14597-1-git-send-email-felipe.balbi@nokia.com>


Fix the following sparse warning:

drivers/usb/core/usb.c:1033:15: warning: symbol 'usb_debug_devices' was not declared. Should it be static?

Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/usb.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/core/usb.c
+++ b/drivers/usb/core/usb.c
@@ -1067,7 +1067,7 @@ static struct notifier_block usb_bus_nb 
 struct dentry *usb_debug_root;
 EXPORT_SYMBOL_GPL(usb_debug_root);
 
-struct dentry *usb_debug_devices;
+static struct dentry *usb_debug_devices;
 
 static int usb_debugfs_init(void)
 {
